### ISSUE TITLE
[Request for a Core Maintainer] - A Public Letter

### ISSUE THREAD

**Issue by matthew-dean** [less/less.js#3148]
_TL;DR version - I'll be stepping down and Less needs a new core maintainer in charge of creating releases, org. management, and repo maintenance. The rest of this is more on the personal side of things, if you're interested._

---
I got involved with Less as a passionate user. Less is a great tool, with an easy learning curve and the accessibility of being a modest superset of CSS.

Since becoming active in contributing to Less on Github issues several years ago, I later helped organize a team to transition the project from the work of the original creator, Alexis Sellier (@cloudhead). There were a lot of people who helped make that possible, probably most significantly the contributions of Luke Page (@lukeapage) who wrote a ton of code for the feature set of what become Less 2.x. Luke also took on the task of releasing, and maintaining build scripts and CI environments.

When Luke started a family and had to step back, I stepped in to help with the organizational side of Less.js. I've done what I can, but on the personal side of things, I'm still in recovery from a serious bout of depression a couple years ago, which had a number of contributing factors, but a major one was: **I was a workaholic and I had too many projects**.

Since that time I've cut out or abandoned more projects than I can count, and Less is one of the ones I knew had to go at some point for my own health. For one, cutting out projects meant I stopped doing web work on the side, which meant I didn't use Less in my free time. At the same time, I started working full time again, at workplaces which hadn't adopted Less.

But I still believe in Less, and I've wanted to get features done for 3.x, which is very close and I still would like to help make happen. But I just wanted to put this out there in case someone was wanting to step up, so that I could help onboard as much as possible while I'm still involved. I don't think I'll stop contributing on Github entirely, but will have a much less active role.

I've also done as much organizational work as possible on doing project boards for future releases (both 3.x and 4.x) with related issues and feature planning.

It's hard when you've put a lot of work into a project to let go of it (believe me, I know), but it's something I need to do and am happy to do what I can to help someone become a point person for Less.

Regards,
Matthew

**Comment by anthony-redFox** [comment]
@matthew-dean I would be happy to help, but guys will be nice if you answer to PRs. I know you busy, will be nice to discuss about direction of project and which are technologys I can use.
So currently I am waiting a answer a lot of time and can't move deeper because you can disapprove my changes but another changes are depend on other PRs.

**Comment by cloudhead** [comment]
Thanks for your amazing work and commitment Matthew :pray:
I posted this on twitter to maybe get a little more eyes on it.

**Comment by matthew-dean** [comment]
@anthony-redFox Which PRs are you waiting on? Also, a lot of the direction / next-steps, planned features are at https://github.com/less/less-meta. Those are kept separate from Less.js Github issues to keep the issues here more focused on bugs and community requests related to current features. So `less-meta` is more "project planning". You'll see there a lot more info on the `3.x` and `4.x` roadmaps.

@cloudhead Thanks so much! And thanks for giving me the opportunity to be so involved in an important project! I also think it's important to acknowledge I'm just one of several people who've given lots of time, especially @lukeapage and @seven-phases-max, but there are others who've done a lot of work on Less.

**Comment by thorn0** [comment]
@matthew-dean Did you consider looking for sponsors instead? Like webpack, babel and other guys do?

**Comment by matthew-dean** [comment]
@thorn0 No. There's nothing wrong with the idea, but even if Less had a sponsor, I have a full time job, so it's not something I could really spend more time on, even if someone was paying me.
